rio/Cargo.toml

```diff
@@ -68,14 +68,14 @@ wayland = [
 ]
 
 [package.metadata.deb]
-name = "rio"
+name = "rio-term"
 maintainer = "Raphael Amorim <rapha850@gmail.com>"
 copyright = "2023, Raphael Amorim <rapha850@gmail.com>"
 depends = "$auto"
 section = "admin"
 priority = "optional"
 assets = [
-    ["../target/release/rio", "usr/local/bin/", "755"],
+    ["../target/release/rio-term", "usr/local/bin/", "755"],
     ["../misc/rio.desktop", "usr/share/applications/", "644"],
```

Comment:
Have you test it in the debian?
wonder if needs to be `rio-term.desktop`

Reply:
I'm getting this error, and I can't solve it:
`cargo-deb: Asset file path does not match any files: /home/daniel/git-src/dev/rio/rio/../target/release/rio-term`
I've change the assests names, changing only the "name" property doens't work when installing the package, the system still install as "rio".

Reply:
`target/release/rio-term` and `rio.desktop` does work then?

Reply:
No, `target/release/rio-term` and` rio.desktop `or `rio-term.desktop` doesn't build at all, getting that path error thing.
It only builds if only change
``` toml
[package.metadata.deb]
name = "rio-term"

```

But this doesn't fix the apt naming conflict, still build as rio